Cargo.toml

```diff
@@ -19,7 +19,7 @@ chrono = { version = "0.4.35", default-features = false, features = [
 chrono-tz = { version = "0.8.4", default-features = false }
 futures-channel = "0.3.29"
 futures-util = { version = "0.3.29", default-features = false }
-http = "1.0.0"
+http = "0.2.0"
```

Comment:
I am curious about this change. What's the rationale? The ecosystem has been slowly moving over to http 1.0. Is there a reason to go back in this crate's case?

Reply:
@jakubadamw This is work in progress, most likely its just some part of the merge that didn't go well.

And that one is easy to find and fix. Thanks!

The version of the fork was way behind the main, so it was always a likely painful process.

Reply:
Yep, there were a number of crates using `0.2` as a dep that would have needed major version bumps to move to `1.0`. I can try to do that as part of this PR, but I was mostly just trying to get the merge done.

Reply:
Ah, I see, fair enough!